### PR TITLE
Rename exposed addressbook names to frienddex

### LINKED ISSUE #85: Rename all addressbook to friendDex
Rename `addressbook` to `friendDex` in the whole app including
- data files
- docs

### PR DESCRIPTION
**Pull request by ivantjh**
Fix #85

### CODE REVIEW

File: src/main/java/seedu/address/model/UserPrefs.java [hunk 1]
```diff
@@ -50,12 +50,12 @@ public void setGuiSettings(GuiSettings guiSettings) {
     }
 
     public Path getAddressBookFilePath() {
-        return addressBookFilePath;
+        return friendDexFilePath;
     }
 
     public void setAddressBookFilePath(Path addressBookFilePath) {
```
Comment (lue97) [inline src/main/java/seedu/address/model/UserPrefs.java:56]: Change method param to `friendDexFilePath`

**Review by lue97 — APPROVED** [review]
lgtm. 
Should we rename the other classes as well? For instance
`model.AddressBook` and `model.ReadOnlyAddressbook`?
There are still other tokens in source code that contains addressbook

**Comment by ivantjh** [comment]
Nope, its quite close to the submission date so I would prefer not to change all the names at one go. The interface of `userPrefs` will still work as per normal and I think we are all used to calling the internal structure that stores `Person` as `addressBook`.

**Review by chesterhow — APPROVED** [review]
LGTM

**Pull request merged**